src/sentry/static/sentry/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -196,12 +197,12 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
     const state = this.state;
     const errors = state.errors;
 
-    // TODO(mark) Figure out how to get measurement keys here.
-    const fieldOptions = generateFieldOptions({
-      organization,
-      tagKeys: Object.values(tags).map(({key}) => key),
-      measurementKeys: [],
-    });
+    const fieldOptions = (measurementKeys: string[]) =>
```

Comment:
Can this be moved inside of `<Measurements>` to simplify this a little?

Reply:
I'll see if I can do this in a follow up PR.